src/pages/index.js

```diff
@@ -117,32 +165,92 @@ function handleCloseModal(modal) {
   disableEsc();
 }
 
+function returnToSave(button) {
+  setTimeout(() => {
+    button.textContent = "Save";
+  }, 2000);
+}
+
 function handleProfileFormSubmit(evt) {
   evt.preventDefault();
-  handleCloseModal(profileFormElement);
+  profileSubmitButton.textContent = "Saving...";
+  const patch = { name: nameInput.value, about: jobInput.value };
+  api
+    .editUserInfo(patch)
+    .then((data) => {
+      profileNameElement.textContent = data.name;
+      profileJobElement.textContent = data.about;
+      handleCloseModal(profileFormElement);
+      returnToSave(profileSubmitButton);
```

Comment:
`      returnToSave(profileSubmitButton);
`

You need to **only** return the **default** button text in the `finally` block, so that the button could get the text regardless of whether the request succeeds or fails. 



**Please, correct it everywhere in the code**